### ISSUE TITLE
Wrong TLS certificate for the naked domain

### ISSUE THREAD

**Issue by flavio** [kubewarden/kubewarden.io#9]
The https://kubewarden.io domain is using a wrong TLS certificate.

Steps to reproduce:
* Go to https://kubewarden.io

-> The browser will report the tls certificate is invalid for the given domain. The TLS certificate used is a generic one from github

**Comment by ereslibre** [comment]
This is clearly a strange situation according to what I can read and I don't really know what is expected to work and what not. Looking at our certificate expiration on www.kubewarden.io I am pretty sure this started to be an issue due to a LetsEncrypt certificate renewal.

**Comment by ereslibre** [comment]
More info: https://stackoverflow.com/questions/54817253/github-pages-https-www-redirect

**Comment by ereslibre** [comment]
I fixed this by setting the CNAME to `kubewarden.io` instead of `www.kubewarden.io`. I have to admit that the GitHub documentation is not very clear right now about all this: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages#using-an-apex-domain-for-your-github-pages-site.

Quoting the documentation as of today:

```
If you are using an apex domain as your custom domain, we recommend also setting up a www subdomain. If you configure the correct records for each domain type through your DNS provider, GitHub Pages will automatically create redirects between the domains. For example, if you configure www.example.com as the custom domain for your site, and you have GitHub Pages DNS records set up for the apex and www domains, then example.com will redirect to www.example.com. Note that automatic redirects only apply to the www subdomain
```

We had everything set up as the last bit says, and our domain set as `www.kubewarden.io`, but it stopped working at some point. I think when the Let's Encrypt certificate was renewed. In any case, I have set `kubewarden.io`, and everything is fine: `kubewarden.io` serves the content, and `www.kubewarden.io` redirects to `kubewarden.io`. All this behaviors are completely dependent on the GitHub redirect implementation side, so we rely completely on their documentation or ont he observed behaviors.

I'm closing this for now because it seems to be working at this time.
